classes/ValidFormBuilder/MultiField.php

```diff
@@ -147,6 +148,8 @@ public function __construct($label, $meta = array())
 
         $this->__dynamic = $this->getMeta("dynamic", $this->__dynamic);
         $this->__dynamicLabel = $this->getMeta("dynamicLabel", $this->__dynamicLabel);
+
+        $this->initialiseDynamicRemoveMeta();
```

Comment:
Yes, I think adding the removable logic to the base class is the best solution for now. It is directly depending on the dynamic field logic, so tying it in with the same context is only natural. There are no dynamic fields that cannot be removed.